compile error

Thanks for putting this package together.  

I ran into a compile error on kinetic:

This is the tail end of the message.  The start scrolls off the terminal:
/home/matt/ros/mbplanner_ws/src/sim/rotors_simulator/rotors_gazebo_plugins/src/gazebo_odometry_plugin.cpp:320:9: error: ‘gazebo_pose’ was not declared in this scope
         gazebo_pose.pos.x);
         ^
/home/matt/ros/mbplanner_ws/src/sim/rotors_simulator/rotors_gazebo_plugins/src/gazebo_odometry_plugin.cpp:336:9: error: ‘gazebo_linear_velocity’ was not declared in this scope
         gazebo_linear_velocity.x);
         ^
/home/matt/ros/mbplanner_ws/src/sim/rotors_simulator/rotors_gazebo_plugins/src/gazebo_odometry_plugin.cpp:343:9: error: ‘gazebo_angular_velocity’ was not declared in this scope
         gazebo_angular_velocity.x);
         ^
make[2]: *** [CMakeFiles/rotors_gazebo_gps_plugin.dir/src/gazebo_gps_plugin.cpp.o] Error 1
make[1]: *** [CMakeFiles/rotors_gazebo_gps_plugin.dir/all] Error 2
/home/matt/ros/mbplanner_ws/src/sim/rotors_simulator/rotors_gazebo_plugins/src/gazebo_ros_interface_plugin.cpp: In destructor ‘virtual gazebo::GazeboRosInterfacePlugin::~GazeboRosInterfacePlugin()’:
/home/matt/ros/mbplanner_ws/src/sim/rotors_simulator/rotors_gazebo_plugins/src/gazebo_ros_interface_plugin.cpp:36:3: error: ‘DisconnectWorldUpdateBegin’ is not a member of ‘gazebo::event::Events’
   event::Events::DisconnectWorldUpdateBegin(updateConnection_);
   ^
make[2]: *** [CMakeFiles/rotors_gazebo_controller_interface.dir/src/gazebo_controller_interface.cpp.o] Error 1
make[1]: *** [CMakeFiles/rotors_gazebo_controller_interface.dir/all] Error 2
make[2]: *** [CMakeFiles/rotors_gazebo_lidar_plugin.dir/src/external/gazebo_lidar_plugin.cpp.o] Error 1
make[1]: *** [CMakeFiles/rotors_gazebo_lidar_plugin.dir/all] Error 2
make[2]: *** [CMakeFiles/rotors_gazebo_odometry_plugin.dir/src/gazebo_odometry_plugin.cpp.o] Error 1
make[1]: *** [CMakeFiles/rotors_gazebo_odometry_plugin.dir/all] Error 2
make[2]: *** [CMakeFiles/rotors_gazebo_ros_interface_plugin.dir/src/gazebo_ros_interface_plugin.cpp.o] Error 1
make[1]: *** [CMakeFiles/rotors_gazebo_ros_interface_plugin.dir/all] Error 2
make: *** [all] Error 2

Am I missing a required package?

--- a/README.md
+++ b/README.md
@@ -10,10 +10,17 @@ git clone https://github.com/unr-arl/mbplanner_ws.git
 
 ## Setup the workspace
 ### Ubuntu 16.04 with ROS Kinetic:  
+#### If gazebo 7.x or 8.x is used:
 Use the ```master``` branch
 ```bash
 cd mbplanner_ws
 ```
+#### If gazebo 9.x is used:
+Use the ```melodic-devel``` branch  
+```bash
+cd mbplanner_ws
+git checkout melodic-devel
+```  
 ### Ubuntu 18.04 with ROS Melodic:  
 Use the ```melodic-devel``` branch  
 ```bash

